101-asp-net-app-service/GettingStarted.md

```diff
@@ -0,0 +1,159 @@
+
+Like this names has to be unique, we will ensure that by adding the name of the resource group where we will deploy the App Service to the name field. The format will be like follows:
+
+```
+<appServiceName>-<resourceGroupName>
+```
+
+#### Variables
+
+The variables are fields that we should not modify. That's because it is configured to get our App Service deployed into a Free tier.
+
+On that template, we have the following variables:
+
+|**VARIABLES NAME**   |**DESCRIPTION**   |
+|---|---|
+|subscriptionId   |ID of our subscription   |
+|hostingEnvironment   |Name of the App Service Environment. If you don't know if you need it, you should leave it empty. Here you can see some [documentation](https://docs.microsoft.com/en-in/azure/app-service/environment/intro)   |
+|serverFarmResourceGroup   |Name of the resource group where our serverFarm is.   |
+|alwaysOn   |It allows us to have the app On even if it is no traffic.   |
+|sku   |Shape for our product.   |
+|skuCode   |Code to identify our product.   |
+|workerSize   |Optional. The worker size. Possible values are Small, Medium, and Large. For JSON, the equivalents are 0 = Small, 1 = Medium, and 2 = Large   |
+|workerSizeId   |Gets or sets size ID of machines: 0 - Small 1 - Medium 2 - Large   |
+|numberOfWorkers   |Gets or sets number of workers.   |
+|hostingPlanName   |Name for the hosting plan. On free tier, you can only have 1 linux hosting environment.   |
+|currentStack   |Framework used on the hosting   |
+|netFrameworkVersion   |Framework version   |
+
+
+#### Resources
+
+The resources are the services that we will to deploy into Azure. In this template we will deploy three resources:
+
+|**RESOURCE NAME**   |**DESCRIPTION**   |
+|---|---|
+|Microsoft.Web/sites   |This is our App Service.   |
+|Microsoft.Web/serverfarms   |This is the hosting for our App Service.  |
+|microsoft.insights/components   |Used to monitor our live web application   |
+
+## Deployment
+
+There are a few ways to deploy this template.
+You can use [PowerShell](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-deploy), [Azure CLI](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-deploy-cli), [Azure Portal](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-deploy-portal) or your favorite SDK.
+
+For this task, we will deploy using the Portal and Azure CLI, I choose to use the Visual Code with Azure CLI extensions, if you like, you can find more information [here](https://code.visualstudio.com/docs/azure/extensions). But bare in mind that you don't need to use the Visual Code, you can stick with the old good always present **Command Line** on Windows or any **bash terminal**.
+
+### Using Azure CLI with Visual Code
+Type on the terminal windows: 
+
+```
+az login
+```
+
+![Screen](./images/az-log.png)
+
+
+You gonna be redirected to the Azure Portal where you can use your credentials to login into.

```

Comment:
they don't actually go to the portal... just a browser

Reply:
Updated